src/index.ts

```diff
@@ -147,12 +154,11 @@ async function bundleMDX(
 
           switch (fileType) {
             case 'mdx': {
-              // I do not want to take the time to type mdx...
-              // eslint-disable-next-line
-              const result = (await createCompiler({
-                remarkPlugins,
-              }).process(contents)) as {contents: string}
-              return {contents: result.contents, loader: 'jsx'}
+              const vfile = await compileMDX(
+                {path: filePath, contents},
+                {jsx: false, remarkPlugins},
```

Comment:
```suggestion
                {jsx: true, remarkPlugins},
```

Set `jsx` to `true` to **keep** JSX. The default is to compile JSX to JS

Reply:
Thanks! Fixed 👍